nss-evening-cohort-14/SweetTooth
# Cart skeleton

## Description
- when user signs in or an account is created, the user from the database is returned
- orderITems of an order can be seen on the cart page 
- GetUserByFirebaseId returns user, addresses, paymentMethods


## Related Issue
#68 





## Motivation and Context
Essential feature of the cart page


## How Can This Be Tested?
fetch branch 
look at cart page 
check in swagger that GetUser with firebaseId returns the addresses and payment methods


## Types of changes
